docs/layouts/partials/prepend-body.html

```diff
@@ -0,0 +1 @@
+<!-- This is deliberately empty to override something we don't want from the theme. -->
```

Comment:
This overrides the file of the same name found in `themes/...`

The original uses a `<noscript>` element to activate Google Analytics in an invisible <iframe> when JS is disabled in the user's browser.

We don't want that because we don't want to activate Google Analytics unless the user explicitly consents to the use of tracking cookies, and we cannot gather that consent without JS enabled, so we simply have to live without GA in this case.